apps/api/src/app.module.ts

```diff
@@ -1,40 +1,22 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
-import { ConfigModule, ConfigService } from '@nestjs/config';
-import { ClientProxyFactory, Transport } from '@nestjs/microservices';
+import { ConfigModule } from '@nestjs/config';
+import { SharedModule } from '@app/shared';
 
 @Module({
   imports: [
     ConfigModule.forRoot({
       isGlobal: true,
       envFilePath: './.env',
     }),
+    SharedModule.registerRmq('AUTH_SERVICE', process.env.RABBITMQ_AUTH_QUEUE),
+    SharedModule.registerRmq(
+      'PRESENCE_SERVICE',
+      process.env.RABBITMQ_PRESENCE_QUEUE,
+    ),
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider extracting RabbitMQ queue names</b></div>
<div id="fix">

Consider extracting the RabbitMQ queue names into constants or configuration variables. This could improve maintainability and make it easier to manage these values across the application.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,6 +1,9 @@
  import { Module } from '@nestjs/common';
  import { AppController } from './app.controller';
  import { AppService } from './app.service';
  import { ConfigModule } from '@nestjs/config';
  import { SharedModule } from '@app/shared';
 
 +const AUTH_QUEUE = process.env.RABBITMQ_AUTH_QUEUE;
 +const PRESENCE_QUEUE = process.env.RABBITMQ_PRESENCE_QUEUE;
 +
  @Module({
    imports: [
      ConfigModule.forRoot({
        isGlobal: true,
        envFilePath: './.env',
      }),
 -    SharedModule.registerRmq('AUTH_SERVICE', process.env.RABBITMQ_AUTH_QUEUE),
 +    SharedModule.registerRmq('AUTH_SERVICE', AUTH_QUEUE),
      SharedModule.registerRmq(
        'PRESENCE_SERVICE',
 -      process.env.RABBITMQ_PRESENCE_QUEUE,
 +      PRESENCE_QUEUE,
      ),
    ],
    controllers: [AppController],
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/6#issuecomment-2448418554>#48ef5b</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged